src/parser/mod.rs

```diff
@@ -12579,7 +12599,9 @@ impl<'a> Parser<'a> {
                     .parse_one_of_keywords(&[Keyword::FROM, Keyword::IN])
                     .is_some()
                 {
-                    parent_name.0.insert(0, self.parse_identifier(false)?);
+                    parent_name
+                        .0
+                        .insert(0, self.parse_identifier(false)?.to_object_name_part());
```

Comment:
Based on this one usage (and I don't see another one in your snowflake PR), couldn't `to_object_name_part` just take ownership and rewrap instead of cloning?

Reply:
You are right. I actually was planning to remove it but forgot. I thought I would use it more